sentry-android-core/src/main/java/io/sentry/android/core/LifecycleWatcher.java

```diff
@@ -81,6 +105,7 @@ public void run() {
   private void cancelTask() {
     if (timerTask != null) {
```

Comment:
Could `cancelTask` concurrently? I guess that'd depend if `onStop` could be called concurrently.
If that's the case (I assume not but), this could be racey now that we set it to `null` below

Reply:
if they do, it'd be a bug in `androidx.lifecycle:lifecycle-process`, but don't think so, `onStop` is a post background callback, we are safe